gradoop-flink/src/main/java/org/gradoop/flink/io/impl/dot/functions/DOTFileFormat.java

```diff
@@ -15,6 +15,8 @@
  */
 package org.gradoop.flink.io.impl.dot.functions;
 
+import  static org.apache.commons.lang3.StringEscapeUtils.escapeHtml4;
```

Comment:
According to my hex editor it is an ordinary 0x20 space character.
![image](https://user-images.githubusercontent.com/36397/29106768-1eaf4546-7cd7-11e7-8b20-6e9bd4066f4a.png)